packages/components/src/form-token-field/test/index.tsx

```diff
@@ -1516,15 +1545,7 @@ describe( 'FormTokenField', () => {
 				'a&nbsp;&nbsp;&nbsp;b',
 				'i&nbsp;&lt;3&nbsp;tags',
 				'1&amp;2&amp;3&amp;4',
-			].forEach( ( tokenHtml ) => {
-				screen.getByText( ( _, node: Element | null ) => {
-					if ( node === null ) {
-						return false;
-					}
-
-					return node.innerHTML === tokenHtml;
-				} );
-			} );
+			].forEach( ( tokenHtml ) => expectEscapedProperly( tokenHtml ) );
```

Comment:
We could shorten this to:

```suggestion
			].forEach( expectEscapedProperly );
```

however this will fail the linter, since it specifically aims to see the invocation of an assertion function.

So I'm going with the spread out version intentionally.